exercise_1/say_hello_spec.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'say_hello'
+
+describe '#greeting' do
+  it 'should output a personalized greeting' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.